examples/integration-tutorials/using-the-analytics-dashboard/using_the_analytics_dashboard.bal

```diff
@@ -0,0 +1,244 @@
+
+                if(response is http:Response) {
+                    io:println("\nGet Patient Record Endpoint");
+
+                    jsonPayload = response.getJsonPayload();
+
+                    if(jsonPayload is json) {
+                        io:println("Patient Record : " + jsonPayload.toString());
+                    } else {
+                        io:println("\n Error : Recieved invalid payload");
+                    }
+                } else {
+                    respondWithError(caller, <string>response.detail().message, "Invalid response recieved");
+                }
+
+                //Update patient record
+                json updatePatient = {
+	                "symptoms":["Cough"],
+	                "treatments":["Panadol"],
+                    "ssn":"234-23-525" 
+                };
+                response = hospitalServiceEP->post("/patient/updaterecord",updatePatient);
+
+                if(response is http:Response) {
+                    io:println("\nUpdate Patient Record Endpoint");
+
+                    textPayload = response.getTextPayload();
+                    
+                    if(textPayload is string) {
+                        io:println(textPayload);
+                    } else {
+                        io:println("\nError : Recieved invalid payload");
+                    } 
+                } else {
+                    respondWithError(caller, <string>response.detail().message, "Invalid response recieved");
+                }
+
+                //Is eligible for discount 
+                response = hospitalServiceEP->get("/patient/appointment/" + untaint appointmentNumber + "/discount");
+
+                if(response is http:Response) {
+                    io:println("\nIs Eligible for Discount Endpoint");
+
+                    textPayload = response.getTextPayload();
+
+                    if(textPayload is string) {
+                        io:println(textPayload);
+                    } else {
+                        io:println("\nError : Recieved invalid payload");
+                    }
+                } else {
+                    respondWithError(caller, <string>response.detail().message, "Invalid response recieved");
+                }
+
+                //Settle payment
+                response = healthcareServiceEP->post("/payments", untaint reservationPayload);
+
+                if(response is http:Response) {
+                    io:println("\n Settle Payment Endpoint");
+
+                    textPayload = response.getTextPayload();
+
+                    if(textPayload is string) {
+                        io:println(textPayload);
+                    } else {
+                        io:println("\nError : Recieved invalid payload");
+                    }
+                } else {
+                    respondWithError(caller, <string>response.detail().message, "Invalid response recieved");
+                }
+
+                //TODO : Settle payment details
```

Comment:
Settle payment details endpoint requires a payment ID to be sent to the endpoint. However, the defined Healthcare service does not return the ID as a JSON payload. Ideally, the Healthcare service should return the ID as a payload that we can access.  

I kept it todo since I cannot access the ID. Should we modify the Healthcare service so that it returns the ID properly?

Reply:
@pramodya1994 Is there a way to return the payment ID or another way to cover this use case?

Reply:
Created an issue to return a JSON response rather than returning a string response. 
Issue: https://github.com/wso2/ballerina-integrator/issues/79